docs/faqs.md

```diff
@@ -0,0 +1,23 @@
+# FAQs #
+
+##Cloning questions##
```

Comment:
The word `questions` is redundant as we are already in FAQ page.  Instead, the name should match the role name for ease of use.

Maybe you can just name this as `satellite-clone`